fix(layout): fix page footer

tikatrsk opened the pull request:
* **fix(layout): fix page footer links and theme switch accessibility**
Improved the page footer by relocating hard-coded links to the 'footerLinks' constant, enhancing accessibility with an aria-label for the icon-only button, and optimizing the ThemeSwitchLabel placement by moving it to the RadioGroupItem. These changes enhance maintainability and user experience.

* **fix(layout): delete unnecessary button components**
The Button component was removed from the link items as it was unnecessary and causing styling and layout issues. This change simplifies the code and improves the overall layout consistency of the page footer.

tikatrsk committed:
refactor(layout): Improve page footer and theme switcher accessibility

 - Move hard-coded links to the 'footerLinks' constant
 - Add aria-label to button with icon only for accessibility
 - Move sr-only ThemeSwitchLabel to RadioGroupItem

triskacode commented on components/page-footer.tsx:
```diff
@@ -28,53 +75,49 @@ export function PageFooter({
+            <div className="font-semibold">{title}</div>
+            <ul className="flex flex-col gap-2">
+              {links.map(({ text, href, target }) => (
+                <li key={text}>
+                  <Button
+                    variant="link"
+                    className="justify-start px-0 text-sm hover:no-underline"
+                    asChild
+                  >
+                    <Link href={href} target={target}>
+                      {text}
+                    </Link>
+                  </Button>
+                </li>
```
Remove unnecesary `Button` component
```suggestion
                <li key={text}>
                    <Link href={href} target={target}>
                      {text}
                    </Link>
                </li>
```

tikatrsk committed:
fix(layout): Delete unnecessary Button components

The Button component was not necessary for the link items.
It was causing unnecessary styling and layout issues.

Co-authored-by: Triska Mahfud K <triska.triskacode@gmail.com>
